### ISSUE TITLE
Source is missing for new deploymen

### ISSUE THREAD

**Issue by asyncee** [pklauke/pycamunda#13]
Hi! First of all thank you for this library.

I was playing with new deployments and have noticed that new deployments are created with `source` field set to `null`.

Code is like this:

```
import logging
from pycamunda import deployment

logging.basicConfig(level=logging.DEBUG)

url = "http://localhost:8080/engine-rest"

create = deployment.Create(url, name="deployment", tenant_id="123", source="dep.bpmn")
create.add_resource(open("diagram_1.bpmn", "rb"))
response = create()
print(response)
```

Deployment is created, file is uploaded and diagram is visible in UI, but the `source` and `tenant` fields are set to `null`: 

![image](https://user-images.githubusercontent.com/701191/94736306-ad173100-0374-11eb-911b-f21d426d957d.png)

Body parameters are (excluding file):

```
{'deployment-name': 'deployment', 'enable-duplicate-filtering': False, 'deploy-changed-only': False, 'deployment-source': 'dep.bpmn', 'tenant-id': '123'}
```

Also, the `/deployment/create` endpoint returns `DeploymentWithDefinitions` object but the library does not return anything, is this intentional? [Reference](https://docs.camunda.org/manual/latest/reference/rest/deployment/post-deployment/).

And last question — is this project production ready?

Thanks!

**Comment by pklauke** [comment]
Thank you very much for pointing this issue out! I will investigate it soon.

Regarding the production-readiness: The package is still under development and does not cover all endpoints of the Camunda REST api. However, a lot of effort is spent in testing the package to catch a good portion of errors.

**Comment by asyncee** [comment]
I have fixed some issues in the library and battle tested it in production. You can check it in my fork, and if you are interested i can do a PR.

**Comment by pklauke** [comment]
Thanks for offering your support. However, I created a fix with consistent serialized response return type, unit tests and docs with the rest of the package. This fix is available in version v0.0.2.

I wasn't able to fix the bug regarding the missing `source`and `tenant_id` so far. Camunda seems to expect these parameters not provided in the request body but in some other way.

**Comment by asyncee** [comment]
Camunda expects body parameters to be form data, instead of json, you can check it out here: https://github.com/asyncee/pycamunda/commit/52b4813e321059e36322441e695850f182858848

**Comment by asyncee** [comment]
Also it would be great if you can apply this fix: https://github.com/asyncee/pycamunda/commit/f4834d224ff99fcf80874efeaedf68a8a2efa926

Without it the most recent camunda version does not display any diagrams, so it expects for resources to have *.bpmn extension, i guess.

**Comment by pklauke** [comment]
Thank you very much for your help! The fix is available in version v0.0.3.

Regarding your issue with the *.bpmn extension: Yes it expects bpmn files to have the extension *.bpmn. It cannot be assumed that every file added is a bpmn file. One can also upload other files. For example *.dmn files. Actually, I also cannot reproduce that your proposed change fixes the problem of the missing diagram display in the Camunda Cockpit.

I am closing this issue assuming that all problems are fixed.
